frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -253,6 +266,9 @@ public void pollBatch(String threadId, long waitTime, Map<String, Job> jobsRepo)
         }
 
         try {
+            if (isUseJobTicket()) {
+                incNumJobTickets();
```

Comment:
Will a similar update to job ticket count be required in the code segment above:
```
        LinkedBlockingDeque<Job> jobsQueue = jobsDb.get(threadId);
        if (jobsQueue != null && !jobsQueue.isEmpty()) {
            Job j = jobsQueue.poll(waitTime, TimeUnit.MILLISECONDS);
            if (j != null) {
                jobsRepo.put(j.getJobId(), j);
                return;
            }
        }
```

Reply:
why?

Reply:
Looks like we are only tracking requests on the `DEFAULT_DATA_QUEUE`, so it won't be required for the above case, makes sense.